subprojects/docs/src/docs/userguide/nativeSoftware.adoc

```diff
@@ -863,7 +863,7 @@ Due to this mechanism, your CUnit sources may not contain a `main` method since
 A api:org.gradle.nativeplatform.test.cunit.CUnitTestSuiteSpec[] component has an associated api:org.gradle.nativeplatform.NativeExecutableSpec[] or api:org.gradle.nativeplatform.NativeLibrarySpec[] component. For each api:org.gradle.nativeplatform.NativeBinarySpec[] configured for the main component, a matching api:org.gradle.nativeplatform.test.cunit.CUnitTestSuiteBinarySpec[] will be configured on the test suite component. These test suite binaries can be configured in a similar way to any other binary instance:
 
 ++++
-<sample id="cunitSources" dir="native-binaries/cunit" title="Registering CUnit tests">
+<sample id="cunitSources" dir="native-binaries/cunit" title="Configuring CUnit tests">
```

Comment:
I think `Configuring` is better here.

Reply:
Yes, I thought it was configuring a build, but looking at it again, it's like JUnit runner.

Reply:
I'm sure it's configuring the tests, so let's keep it like this.